integration-web/src/main/java/org/symphonyoss/integration/web/properties/AuthErrorMessageKeys.java

```diff
@@ -0,0 +1,17 @@
+package org.symphonyoss.integration.web.properties;
+
+import org.symphonyoss.integration.web.resource.ApplicationAuthenticationResource;
+import org.symphonyoss.integration.web.resource.ApplicationAuthorizationResource;
+
+/**
+ * Exception message keys used by the components {@link ApplicationAuthenticationResource} and
+ * {@link ApplicationAuthorizationResource}
+ *
+ * Created by rsanchez on 28/08/17.
```

Comment:
Is this the correct date format?

Reply:
@aurojr I'm not sure what's the correct one. I would say there is no standard, I'm using 'dd/MM/yy' in all of them.

Would you suggest to move to 'MM/dd/yyyy'?

Reply:
If there's no standard, there's no reason to change that :-)